splitio/models/telemetry.py

```diff
@@ -68,6 +67,25 @@
 TREATMENT_WITH_CONFIG = 'treatmentWithConfig'
 TREATMENTS_WITH_CONFIG = 'treatmentsWithConfig'
 TRACK = 'track'
+CONNECTION_ESTABLISHED = 'CONNECTION_ESTABLISHED'
```

Comment:
given the large amounts of constants here, we could consider using enums to keep them scoped and more legible.
For example:

```
from enum import Enum

class ApiOperations(Enum):
  SPLIT_SYNC = 0
  SEGMENT_SYNC = 1
  ...


class ClientMethods(Enum):
  GET_TREATMENT = 0
  GET_TREATMENTS = 1
  ...
```

from other modules we would only import the enum class and use its values scoped:
```
from ...... import ClientMethods

record(ClientMethods.GET_TREATMENT, .....)
```

and when comparing within this module:

```
if method == ClientMethods.GET_TREATMENT:
   ...
elif method == ClientMethods.GET_TREATMENTS:
  ...
```

Reply:
Will do